homeassistant/components/airly/config_flow.py

```diff
@@ -16,11 +16,7 @@
 from homeassistant.helpers.aiohttp_client import async_get_clientsession
 import homeassistant.helpers.config_validation as cv
 
-from .const import (  # pylint:disable=unused-import
-    CONF_USE_NEAREST,
-    DOMAIN,
-    NO_AIRLY_SENSORS,
-)
+from .const import CONF_USE_NEAREST, DOMAIN, NO_AIRLY_SENSORS
```

Comment:
```suggestion
from .const import (  # pylint:disable=unused-import
    CONF_USE_NEAREST,
    DOMAIN,
    NO_AIRLY_SENSORS,
)
```

Reply:
But why? we have no unused imports here 🤔

Reply:
Oh I see, sorry about that